modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js

```diff
@@ -120,5 +120,10 @@ export function populateConfigData([
 		tableView: tableViews.find(({id}) => id === app.dataListViewId),
 	};
 
+	if (app.active && config.formView.missingRequiredFields.nativeField) {
+		config.formView = {};
```

Comment:
Hey @jotascript , Do we need to use the optional chaining operator, `config.formView.missingRequiredFields.?nativeField` like you did [here](https://github.com/liferay-workflow/liferay-portal/pull/252/commits/d263c781c8a68c695e2f199777d745fa87002333#diff-479bc0418b3a6e2682d7b2d71a68e0d81b6d70dc30a06ed179bb583814a57461R105)? What do you think?

Thx

Reply:
Hey @rafaprax , I think for this and others occurrences of this variable the optional chaining is not required, because the object always is created when we are checking all form views.

Reply:
ok then